src/snowflake/cli/app/telemetry.py

```diff
@@ -145,6 +146,17 @@ def log_command_usage():
     _telemetry.send({TelemetryField.KEY_TYPE: TelemetryEvent.CMD_EXECUTION.value})
 
 
+@ignore_exceptions()
+def log_command_execution_error(exception: Exception):
+    exception_type: str = type(exception).__name__
+    _telemetry.send(
+        {
+            TelemetryField.KEY_TYPE: TelemetryEvent.CMD_EXECUTION_ERROR.value,
+            CLITelemetryField.ERROR_TYPE: exception_type,
+        }
+    )
+
+
```

Comment:
This misses execution context like what command was called etc.

Reply:
It doesn't, the data is generated in [generate_telemetry_data_dict](https://github.com/snowflakedb/snowflake-cli/blob/main/src/snowflake/cli/app/telemetry.py#L107), this an example output:
{
    "message": {
        "driver_type": "PythonConnector",
        "driver_version": "3.11.0",
        "source": "snowcli",
        "version_cli": "0.0.0-test_patched",
        "version_os": "macOS-14.5-arm64-arm-64bit",
        "version_python": "3.11.5",
        "config_feature_flags": {"dummy_flag": "True", "wrong_type_flag": "UNKNOWN"},
        "command": ["app", "run"],
        "command_group": "app",
        "command_flags": {
            "validate": "DEFAULT",
            "project_definition": "DEFAULT",
            "diag_log_path": "DEFAULT",
            "format": "DEFAULT",
        },
        "command_output_type": "TABLE",
        "project_definition_version": "1",
        "type": "error_executing_command",
        "error_type": "SourceNotFoundError",
    },
    "timestamp": "1719576630554",
}